src/components/Popover/Popover.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
+import { ReactComponent as Close } from '../../assets/icons/Close.svg';
+import { IconButton } from '../IconButton';
+import { Text } from '../Text';
+import { TPAComponentsConsumer } from '../TPAComponentsConfig';
+import { POPOVER_DATA_HOOKS, POPOVER_DATA_KEYS } from './dataHooks';
+import styles from './Popover.st.css';
+
+export interface PopoverProps {
+  title?: string;
+  onClose(): void;
+  withArrow?: boolean;
+  rightArrow?: boolean;
+  arrowTop?: string;
+  withShadow?: boolean;
+}
+
+interface DefaultProps {
+  title: string;
+  'data-hook': string;
+  withArrow: boolean;
+  rightArrow: boolean;
+  arrowTop: string;
+  withShadow: boolean;
+}
+
+/** Popover */
+export class Popover extends React.Component<PopoverProps> {
+  static displayName = 'Popover';
+  static defaultProps: DefaultProps = {
+    title: '',
+    'data-hook': POPOVER_DATA_HOOKS.Popover,
```

Comment:
as in the other PR's
it's best not to add a default data-hook.
It will encourage consumers to use this instead of adding their own, which means this will be a binding contract, and we prefer to avoid any unnecessary api

Reply:
to keep in the props ?